.github/workflows/render_rmarkdown.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    paths: ['**.Rmd']
+
+name: render-rmarkdown
+
+jobs:
+  render-rmarkdown:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+      
+      - name: Render Rmarkdown files and Commit Results
+        run: |
+          Rscript -e "rmarkdown::render_site()"
+          if [[ "$(git status --porcelain)" != "" ]]; then
+            git config --global user.name "$GITHUB_ACTOR"
+            git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+            git add *
+            git commit -m "Auto update dashboard"
+            git push
+          fi
```

Comment:
Maybe we can have an else which will just `echo` that there were no changes detected? Just for the sake of logging,